### ISSUE TITLE
Fix memory leak for pubsub leaveallrooms event listener

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#316]
### Problem

I noticed this in travis/test output logs:

```
(node:9568) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 leaveallrooms listeners added. Use emitter.setMaxListeners() to increase limit
```

### Solution

Figure out what the deal is and fix it.

**Comment by vijayee** [comment]
This is error occurs anytime an events.EventEmitter exceeds 10 listeners regardless of reason. Its not technically indicative of an actual leak. Do you have the whole output that cause this or how to reproduce it?

**Comment by taoeffect** [comment]
This has something to do with the rooms plugin for Primus on the backend. I don't remember whih Travis build this came from.

**Comment by taoeffect** [comment]
Hopefully closed by #447. Will re-open if it re-appears.
